crates/core/src/client/config.rs

```diff
@@ -161,3 +165,33 @@ impl SequencerGatewayProviderBuilder {
         self.0
     }
 }
+
+pub async fn get_starknet_account_from_env<P: Provider + Send + Sync + 'static>(
+    provider: Arc<P>,
+) -> Result<SingleOwnerAccount<Arc<P>, LocalWallet>> {
+    let (starknet_account_private_key, starknet_account_address) = {
+        let network = get_env_var("STARKNET_NETWORK")?;
```

Comment:
since this PR has lots of logic, let's not remove all get_.... 
but we can create an issue to refactor some util and functions in our codebase to not have "get" keyword

Reply:
I will create the issue ser 🙌